code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -81,81 +100,96 @@
+		/obj/item/encryptionkey/syndicate) // 10TC
+
+/obj/item/storage/box/syndie_kit/bundle/professional // 165TC
+	name = "Sniper Bundle"
+	desc = "Suit up and handle yourself like a professional with a long-distance sniper rifle, additional .50 standard and penetrator rounds and thermal glasses to easily scope out your target."
+	items = list(
+		/obj/item/gun/projectile/automatic/sniper_rifle/syndicate, // 80TC
+		/obj/item/ammo_box/magazine/sniper_rounds, // 15TC
+		/obj/item/ammo_box/magazine/sniper_rounds/penetrator, // 20TC
+		/obj/item/ammo_box/magazine/sniper_rounds/penetrator, // 20TC
+		/obj/item/clothing/glasses/chameleon/thermal, // 15TC
 		/obj/item/clothing/gloves/combat, // 0 TC
 		/obj/item/clothing/under/suit/really_black, // 0 TC
 		/obj/item/clothing/suit/storage/iaa/blackjacket/armored, // 0TC
-		/obj/item/encryptionkey/syndicate) // 2TC
+		/obj/item/encryptionkey/syndicate) // 15TC
 
-	var/static/list/grenadier = list( // 20TC + Tactical Grenadier Belt
+/obj/item/storage/box/syndie_kit/bundle/grenadier // 132TC + Tactical Grenadier Belt
+	name = "Grenadier Bundle"
+	desc = "A variety of grenades and pyrotechnics to ensure you can blast your way through any situation. "
+	items = list(
 		/obj/item/storage/belt/grenade/tactical, // Contains 2 Frag and EMP grenades, 5 C4 Explosives, 5 Smoke and Gluon grenades and 1 Minibomb grenade ~20TC Estimate
-		/obj/item/gun/projectile/automatic/pistol, // 4TC
-		/obj/item/ammo_box/magazine/m10mm/fire, // 2TC
-		/obj/item/ammo_box/magazine/m10mm/fire, // 2TC
-		/obj/item/mod/control/pre_equipped/traitor, // 6TC
+		/obj/item/gun/projectile/automatic/pistol, // 20TC
+		/obj/item/ammo_box/magazine/m10mm/fire, // 6TC
+		/obj/item/ammo_box/magazine/m10mm/fire, // 6TC
+		/obj/item/mod/control/pre_equipped/traitor, // 30TC
 		/obj/item/clothing/gloves/combat, // 0TC
-		/obj/item/card/id/syndicate, // 2TC
-		/obj/item/clothing/shoes/chameleon/noslip, // 2TC
-		/obj/item/encryptionkey/syndicate) // 2TC
-
-	var/static/list/metroid = list( // 21 + modules + laser gun
-		/obj/item/mod/control/pre_equipped/traitor_elite, // 9TC
-		/obj/item/mod/module/visor/thermal, // 3 TC
-		/obj/item/mod/module/stealth, //0 TC but strong
-		/obj/item/mod/module/power_kick, //0 TC but funny
-		/obj/item/mod/module/sphere_transform, //0TC but should not be allowed to normally be installed
-		/obj/item/autosurgeon/organ/syndicate/laser_arm, //0 TC but otherwise not obtainable.
-		/obj/item/pinpointer/advpinpointer, //4 TC
-		/obj/item/storage/box/syndidonkpockets, //2TC, otherwise they will just die in the first combat to disabler.
-		/obj/item/storage/belt/utility/full/multitool, //0 TC
-		/obj/item/clothing/head/collectable/slime,  //Priceless (0 TC)
-		/obj/item/encryptionkey/syndicate) //2 TC
-
-
+		/obj/item/card/id/syndicate, // 10TC
+		/obj/item/clothing/shoes/chameleon/noslip, // 10TC
+		/obj/item/encryptionkey/syndicate) // 10TC
+
+/obj/item/storage/box/syndie_kit/bundle/metroid // 115TC + modules + laser gun
```

Comment:
My god this bundle is cheap

Reply:
this one shows the new TC counts, the other ones are the old TC counts

Reply:
all comments are new prices now, most of those modules still don't have defined TC prices